### PR TITLE
SPCTR-2928: Fix - Container flex properties not applied as intended in frontend when header and image blocks are added header section is taking full height.

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

 Container flex properties are not applied when heading block and image block is added in innercontainer as the header section on the left is reset to full height.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by aapatre — APPROVED** [review]

File: includes/blocks/container/frontend.css.php [hunk 1]
```diff
@@ -364,7 +364,7 @@
 	$m_selectors[ '.uagb-block-' . $id . ' .uagb-container__video-wrap' ] = $border_mobile;
 
 	if ( 'alignwide' === $attr['innerContentWidth'] && 'alignfull' === $attr['contentWidth'] ) {
-		$selectorClass = '.uagb-block-' . $id . ' .uagb-container-inner-blocks-wrap';
+		$selectorClass = '.wp-block-uagb-container';
```
Comment (AbhijitNage123) [inline includes/blocks/container/frontend.css.php:367]: Changing this will break everything? Make sure nothing should break

File: README.md [hunk 2]
```diff
@@ -170,6 +170,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 * Improvement: Global - Added support for Spectra One's preview options.
 * Fix: Table Of Contents - Fixed the improper HTML markup/structure for the heading list.
 * Fix: Testimonial - Resolved an issue in which the default Testimonial blocks would share same the background image.
+* Fix: Container - Resolved flex properties not applied as intended in frontend when header and image blocks are added header section is taking full height.
```
Comment (bsf-zanev) [inline README.md:173]:
```suggestion
* Fix: Container - Resolved an issue in which the flex properties would not work as intended in the frontend for the Image and Heading blocks in nested Containers.
```

**Review by bsf-zanev — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

File: includes/blocks/container/frontend.css.php [hunk 3]
```diff
@@ -369,7 +369,7 @@
...
 	} else {
 		$selectorClass = '.wp-block-uagb-container.uagb-block-' . $id;
```
Comment (bsf-zanev) [inline includes/blocks/container/frontend.css.php:374]: If we're using the same selector here in both cases, do we need the if condition...?

If so, why did we need it? The code seems to be correctly functioning with the main block selector.

Comment (rajmohanreddy774) [reply includes/blocks/container/frontend.css.php]: I have removed if condition and used main block selector it is working fine.

File: includes/blocks/container/frontend.css.php
Comment (bsf-zanev) [inline includes/blocks/container/frontend.css.php:1]: Resolve Code Analysis issue here.

**Review by bsf-zanev — APPROVED** [review]
Just need to rearrange the changelog entry, grunt readme, and fix the small code analysis issue.

Rest all looks good to go.

File: readme.txt [hunk 4]
```diff
@@ -218,6 +218,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 = x.x.x =
 * Fix: Modal - Resolved an issue in which the Modal would not open in Astra's quick-view popup for WooCommerce products.
 * Fix: Separator - Resolved an edge-case issue in which the responsive alignment would not work as intended.
+* Fix: Container - Resolved an issue in which the flex properties would not work as intended in the frontend for the Image and Heading blocks in nested Containers.
```
[the remark this answers is not in the document]
Comment (rajmohanreddy774) [reply readme.txt]: Done.

File: includes/blocks/container/frontend.css.php
Comment (rajmohanreddy774) [reply includes/blocks/container/frontend.css.php:1]: Done.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**